specs/windows/windows_search.table

```diff
@@ -0,0 +1,27 @@
+table_name("windows_search")
+description("Run searches against the Windows system index database using Advanced Query Syntax. See https://learn.microsoft.com/en-us/windows/win32/search/-search-3x-advancedquerysyntax for details.")
+schema([
+    Column("name", TEXT, "The name of the item"),
+    Column("path", TEXT, "The full path of the item."),
+    Column("size", BIGINT, "The item size in bytes."),
+    Column("date_created", TEXT, "The date and time the item was created."),
```

Comment:
I compiled the code to play with the table a bit and found that `date_created` field and `date_modifed` were not working for me.  It could be that the data type for these two columns needs to be [DATETIME](https://github.com/osquery/osquery/blob/9f09b7c632a784ad5493e0fc05ff74a0f89c6efd/specs/windows/process_etw_events.table#L16). This will enable queries such as the one below

```
"select * from windows_search WHERE query = '\"test\" folder:documents' AND date_created >= (( SELECT unix_time FROM time) - 60 );"
"select * from windows_search WHERE query = '\"test\" folder:documents' AND date_created > '2022-11-18 16:48:00';"
"select * from windows_search WHERE query = '\"test\" folder:documents' AND date_created BETWEEN '2022-11-18 16:40:00' AND '2022-11-18 16:50:00';"
```

Reply:
nice catch! I've updated it and successfully ran your queries.

Reply:
.... wait nvm, they are not all working

Reply:
The first issue was that date_created and date_modified were defined at TEXT columns as you pointed out. These values are unix timestamps. So I've updated them to be integers. `datetime()` func can be used to convert to datetime format.

I've updated the descriptions of date_created and date_modified in the spec as well as included these examples.
```sql
select * from windows_search WHERE query = 'folder:documents' AND date_created >= (( SELECT unix_time FROM time) - 60 )
select *, datetime(date_created, 'unixepoch') as datetime from windows_search WHERE query = 'folder:documents' AND datetime > '2022-11-18 16:48:00'
select *, datetime(date_created, 'unixepoch') as datetime from windows_search WHERE query = 'folder:documents' AND datetime BETWEEN '2022-11-18 16:40:00' AND '2023-11-18 16:50:00'
```

Reply:
correction, I update the dates to be integers

Reply:
Thanks for the update @James-Pickett! I'm marking this as resolved now